src/feature/token/index.js

```
@@ -147,9 +148,27 @@ class Token {
+  }
+
+  /**
+   * Theme暴露使用的颜色变量
+   * @returns object
+   */
+  static getColors() {
+
+    const colorHandler = (_, prop) => {
+      const vaild = EXPORT_COLORS_KEY.includes(prop)
+      return vaild ? this.config.exportColors[prop] : undefined
+    }
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)